workers/shared-worker-parse-error-failure.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<title>SharedWorker: parse error failure</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+
+promise_test(async () => {
+  const scriptURL = 'modules/resources/syntax-error.js';
+  const worker = new SharedWorker(scriptURL, { name: 'classic',
```

Comment:
It's not clear why some of these tests have `name` values and some don't. If it's for debugging I'd expect distinct names on all of them. Otherwise I'd just omit it.

Reply:
This is because SharedWorker instances with same name, script url and constructor origin are connected to the same WorkerGlobalScope. As for the first 2 tests, SharedWorker instances have the same script url and constructor origin, so to distinguish them we need to assign the different names here. In this test, it is expected to fail constructing WorkerGlobalScope, so using the same name is not the problem, but it's not nice when the test doesn't run as expected.

Reply:
I see. Maybe just give them all unique names then, even if it's not strictly necessary.

Reply:
After this comment, I decided to separate the test into 'classic' and 'module', so I removed the worker's name anyway.